docs/source/using/experimental-features.adoc

```diff
@@ -73,6 +73,38 @@ If you prefer to use dynamic assignment, you can use:
 $ minishift ip --set-dhcp
 ----
 
+[[local-proxy-server]]
+== Local Proxy Server
+
+{project} provides an option to run a local proxy server on the host, which the {project} instance can use to access the internet.
+This can help in the situation when local security certificates are used in the organization, but can not easily be shared with the instance.
+
+Enabling the proxy server is done using the following command:
+
+----
+$ minishift config set local-proxy true
+----
+
+This will start a proxy server on the host which on start will automatically be assigned to the {project} instance.
+
+When the corporate or upstream proxy is known, you can specify this with the following configuration option:
+
+----
+$ minishift config set upstream-proxy [username:password@host:port]
```

Comment:
It is not clear that I can use a proxy address starting with `http://` or `https://` here.